Model/Eps/GetFastlaneStyles.php

```diff
@@ -38,9 +38,14 @@ public function __construct(GetCommand $getCommand, Config $config)
      * @param int $websiteId
      * @return array
      */
-    public function getStyles(int $websiteId): array
+    public function getStyles(int $websiteId, string $baseUrl): array
```

Comment:
Could you please add this new parameter to the PHPDoc docblock as well?

Reply:
Added.